code/modules/mining/drilling/scanner.dm

```diff
@@ -1,8 +1,8 @@
 /obj/item/weapon/mining_scanner
-	name = "ore detector"
+	name = "Deep Scan Device"
```

Comment:
This isn't a proper noun, should be lowercase.

Reply:
Thank you - bit finicky with that stuff